otelcollector/prometheuscollector/main.go

```diff
@@ -0,0 +1,270 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
```

Comment:
prometheuscollector folder -> main

main.go -> ccpmain.go

package main -> package ccpmain

create a common folder that will be shared across images and create one for the other